allennlp/modules/seq2seq_decoders/simple_decoder_cell.py

```diff
@@ -0,0 +1,121 @@
+import torch
+from typing import Tuple
+
+from torch.nn import LSTMCell
+
+from allennlp.common.checks import ConfigurationError
+from allennlp.modules import Attention, SimilarityFunction
+from allennlp.modules.attention import LegacyAttention
+from allennlp.modules.seq2seq_decoders.decoder_cell import DecoderCell
+from allennlp.nn import util
+
+
+@DecoderCell.register("simple_decoder")
+class SimpleDecoderCell(DecoderCell):
```

Comment:
Probably `AttentionLstmDecoderCell`, or something like that?  Or even just `LstmDecoderCell`, because attention is implied these days, and you can optionally just not pass it?

Reply:
Changed